### PR TITLE
Update dependencies to latest patch

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Update dependencies to latest patch version aver verification via `bin/console security:check`
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Install shop, BO works

## Updated dependencies:

* symfony/symfony: v3.4.26 -> v3.4.36
* twig/twig: v1.38.4 -> v1.42.4
* symfony/phpunit-bridge: v3.4.23 -> v3.4.36

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17139)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
         "symfony/polyfill-apcu": "^1.0",
         "symfony/polyfill-php73": "^1.10",
         "symfony/swiftmailer-bundle": "^2.6.4",
-        "symfony/symfony": ">=3.4.26",
+        "symfony/symfony": "~3.4.26",
```
Comment (PierreRambaud) [inline composer.json]:
```suggestion
        "symfony/symfony": "~3.4.36",
```

File: composer.json [hunk 2]
```diff
@@ -105,7 +105,7 @@
...
         "tecnickcom/tcpdf": "^6.2.12",
         "tijsverkoyen/css-to-inline-styles": "^2.2",
         "twig/twig": "^1.38||^2.6.2"
```
Comment (PierreRambaud) [inline composer.json]:
```suggestion
        "twig/twig": "^1.42||^2.6.2"
```

**Review by PierreRambaud — DISMISSED** [review]

**Comment by eternoendless** [comment]
Looks like tests are failing because of a data type issue. Smells like a breaking change or side effect...

```
1) LegacyTests\Integration\PrestaShopBundle\Controller\Api\StockManagementControllerTest::itShouldReturnOkResponseWhenRequestingProductsStock with data set #0 (array(), 1)
There should be 1 page(s) counting 100 elements at most.
Failed asserting that '1' is identical to 1.
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:221
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:190
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:100
2) LegacyTests\Integration\PrestaShopBundle\Controller\Api\StockManagementControllerTest::itShouldReturnOkResponseWhenRequestingProductsStock with data set #1 (array(1, 2), 24)
There should be 24 page(s) counting 2 elements at most.
Failed asserting that '24' is identical to 24.
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:221
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:190
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:100
3) LegacyTests\Integration\PrestaShopBundle\Controller\Api\StockManagementControllerTest::itShouldReturnOkResponseWhenRequestingProductsStock with data set #4 (array(5, 1, 1), 4)
There should be 4 page(s) counting 1 elements at most.
Failed asserting that '4' is identical to 4.
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:221
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:190
/home/travis/build/PrestaShop/PrestaShop/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php:100
4) LegacyTests\Integration\PrestaShopBundle\Controller\Api\StockManagementControllerTest::itShouldReturnOkResponseWhenRequestingProductsStock with data set #5 (array(array(4, 5), 1, 1), 12)
There should be 12 page(s) counting 1 elements at most.
Failed asserting that '12' is identical to 12.
```

**Comment by atomiix** [comment]
Since `HeaderBag` cast the return value to be a `string`, I had to change the expected values on the appropriate tests.

See [the change in Symfony](https://github.com/symfony/symfony/commit/ef5ead00059fc321e7fda20c2dfdd4440585be26#diff-5f886132f28113d74a1a219e967e88fcR124)

**Comment by PierreRambaud** [comment]
@atomiix You need to cast int here: https://github.com/PrestaShop/PrestaShop/blob/1.7.6.x/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php#L217
Since it's a number, it must be an int, not a string

**Comment by matks** [comment]
I think we can close this PR ? @PierreRambaud

**Comment by PierreRambaud** [comment]
Yes, dependencies has been updated for the 1.7.6.5

**Pull request closed without merging**